plugins/inventory/digital_ocean.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Per-Arne <per@sysx.no>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+import json
+
+from ansible.module_utils.urls import open_url
+
+DOCUMENTATION = r'''
+    name: digitalocean
+    plugin_type: inventory
+    author:
+        - Per-Arne Andersen
+    short_description: Digital ocean inventory source
+    extends_documentation_fragment:
+        - constructed
+    description:
+        - Get inventory hosts from Digital Ocean public cloud.
+        - Uses an YAML configuration file ending with either I(digital_ocean.yml) or I(digital_ocean.yaml) to set parameter values (also see examples).
+        - Uses I(api_config), I(~/.digital_ocean.ini), I(./digital_ocean.ini) or C(DIGITALOCEAN_API_CONFIG) pointing to a Digitalocean credentials INI file
+          (see U(https://docs.ansible.com/ansible/latest/scenario_guides/guide_digitalocean.html)).
+    options:
+        plugin:
+            description: Token that ensures this is a source file for the 'digitalocean' plugin.
+            type: string
+            required: True
+            choices: [ digitalocean ]
+        api_account:
+            description: Specify the account to be used.
+            type: string
+            default: default
+        api_config:
+            description: Path to the Digital Ocean configuration file. If not specified will be taken from regular Digital Ocean configuration.
+            type: path
+            env:
+                - name: DIGITALOCEAN_API_CONFIG
+        api_key:
+            description: Digital Ocean API key. If not specified will be taken from regular Digital Ocean configuration.
+            type: string
+            env:
+                - name: DIGITALOCEAN_API_KEY
+        hostname:
+            description: Field to match the hostname. Note v4_main_ip corresponds to the main_ip field returned from the API and name to label.
+            type: string
+            default: v4_main_ip
+            choices:
+                - v4_main_ip
+                - v6_main_ip
+                - name
+        filter_by_tag:
+            description: Only return servers filtered by this tag
+            type: string
+'''
+
+EXAMPLES = r'''
+# inventory_digitalocean.yml file in YAML format
+# Example command line: ansible-inventory --list -i inventory_digitalocean.yml
+
+# Group by a region as lower case and with prefix e.g. "digitalocean_region_ams3" and by OS without prefix e.g. "CentOS_7_x64"
+plugin: digitalocean
```

Comment:
```suggestion
plugin: digitalocean
```
I am leaning over `digital_ocean`